examples/time_frequency/plot_time_frequency_erds.py

```diff
@@ -111,3 +113,85 @@
     fig.colorbar(axes[0].images[-1], cax=axes[-1])
     fig.suptitle("ERDS ({})".format(event))
     fig.show()
+
+##############################################################################
+# Similar as for :class:`~mne.Epochs` objects, we can also export data from
+# :class:`~mne.time_frequency.EpochsTFR` and
+# `~mne.time_frequency.AverageTFR` objects to a
+# :class:`Pandas DataFrame <pandas.DataFrame>`:
+
+df = tfr.to_data_frame()
+df.time = df.time / 1000  # time to s
+df.head()
+
+##############################################################################
+# This allows us to use additional plotting functions like
+# :func:`seaborn.lineplot` to easily plot confidence bands:
```

Comment:
the nested for-loops in this section can be eliminated by using a long-format dataframe, setting the channel and band columns as categorical, and making use of `sns.FacetGrid`. Here's a starting point:

```py
df = tfr.to_data_frame(time_format=None, long_format=True)
df['channel'].cat.reorder_categories(['C3', 'Cz', 'C4'], ordered=True, inplace=True)

freq_bands = {'delta': (0.5, 4),
              'theta': (5, 7),
              'alpha': (8, 14),
              'beta': (15, 35)}

def map_bands(freq):
    for band, (low_lim, high_lim) in freq_bands.items():
        if low_lim <= freq <= high_lim:
            return band

df['band'] = pd.Categorical(
    df['freq'].map(map_bands),
    categories=['beta', 'alpha', 'theta', 'delta'],
    ordered=True)

g = sns.FacetGrid(df, row='band', col='channel', margin_titles=True)
g.map(sns.lineplot, 'time', 'value', 'condition', n_boot=10)
```

Reply:
Thanks! That looks way better and I implemented it that way.